reana_client/cli/quotas.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client quotas related commands."""
+
+import logging
+import sys
+
+import click
+from reana_client.cli.utils import (
+    add_access_token_options,
+    check_connection,
+    NotRequiredIf,
+)
+
+
+@click.group(help="Quota commands")
+def configuration_group():
```

Comment:
WDYT about calling this group with a more meaningful name? Something like `quota_group`?